coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -92,6 +92,11 @@
         'args': '--new-window',
         'gui': True
     },
+    'code': {
+        'file_arg_template': '{filename}',
```

Comment:
Can we add the line, too?

Reply:
Let's file an issue for this after then :+1:

Reply:
it does have a documented option `-g` https://code.visualstudio.com/docs/editor/command-line

Reply:
oh yeah nice, thx @SanketDG 
@ayush9398 please update the args with `-g` :+1: